pkg/common/utils/utils.go

```diff
@@ -428,3 +436,13 @@ func UpdatePodLabelForAdmissionController(pod *v1.Pod, namespace string) map[str
 
 	return result
 }
+
+func UpdatePodAnnotationForAdmissionController(pod *v1.Pod, key string, value string) map[string]string {
```

Comment:
If this code is only used by the A.C., then "pkg/admission" is a better place for this function.

Reply:
Agreed.

Reply:
Updated in latest PR.